app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -62,9 +62,9 @@ class MainActivity : AppCompatActivity() {
 
             override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) { }
 
-            override fun afterTextChanged(s: Editable?) {
-                if (s?.isNotEmpty() == true) {
-                    var filteredQuery = testMovieData.filter { movie -> movie.title.toLowerCase().contains(s) };
+            override fun afterTextChanged(searchTxt: Editable?) {
+                if (searchTxt?.isNotEmpty() == true) {
+                    var filteredQuery = testMovieData.filter { movie -> movie.title.toLowerCase().contains(searchTxt) };
```

Comment:
For kotlin, and really any language as far as I am aware, variable names are typically a noun or a noun phrase (see https://developer.android.com/kotlin/style-guide#naming_2). While this particular instance is fairly inconsequential, as from this code you can pretty easily understand what "s" is, it still is a good idea to get in the habit of naming variables to be more descriptive.